includes/class-wc-taxjar-integration.php

```diff
@@ -1118,8 +1118,8 @@ public function get_line_items( $wc_cart_object ) {
 		$line_items = array();
 
 		foreach ( $wc_cart_object->get_cart() as $cart_item_key => $cart_item ) {
-			$product = $cart_item['data'];
-			$id = $product->get_id();
+			$id = $cart_item[ 'data' ]->get_id();
```

Comment:
These two lines contain the primary fix for the issue. Due to the way we cause the calculation methods to rerun, and the subscription plugins interaction with them, the tax class of the product was being altered before we performed the API requests. This fix ensures the correct data from the product is used.

Reply:
Code change makes sense to me. Use `wc_get_product` instead of relying on `$cart_item['data']`.